packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -196,5 +202,33 @@ Don't mix 'authors' with other existing 'author_*' front matter. Choose one or t
     return [authorLegacy];
   }
 
+  const inlineAuthors = updatedAuthors.filter((author) => author.inline);
+
+  const duplicateList = updatedAuthors.filter(
+    (author, index, self) =>
+      index !== self.findIndex((t) => t.name === author.name),
+  );
```

Comment:
Author might have no name

What happens if we have 2 authors with just an image? Are they considered as duplicate? 

What happens if one predefined author has key "seb" and name "Seb", and there's another inline author named "Seb", are they considered as duplicate?

The current implementation is not good enough, and the method to detect duplicates is complex enough to deserve its own unit tests

Reply:
This case should work and not warn

https://deploy-preview-10224--docusaurus-2.netlify.app/tests/blog/2022/01/20/image-only-authors/

![CleanShot 2024-06-20 at 16 42 35](https://github.com/facebook/docusaurus/assets/749374/ecfc9872-43bc-4dea-8824-92f333d43984)